xbmc/CompileInfo.h

```diff
@@ -30,5 +30,6 @@ class CCompileInfo
   static const char* GetCopyrightYears();
   static std::string GetBuildDate();
   static const char* GetVersionCode();
+  static const std::vector<std::string> GetAvailableWindowSystems();
```

Comment:
```suggestion
  static std::vector<std::string> GetAvailableWindowSystems();
```

Don't return `const`